README.md

```diff
@@ -60,24 +60,28 @@ npm install @google-cloud/container
 ### Using the client library
 
 ```javascript
-const container = require('@google-cloud/container');
-
-// Create the Cluster Manager Client
-const client = new container.v1.ClusterManagerClient();
-
-async function quickstart() {
-  const zone = 'us-central1-a';
-  const projectId = await client.getProjectId();
-  const request = {
-    projectId: projectId,
-    zone: zone,
-  };
-
-  const [response] = await client.listClusters(request);
-  console.log('Clusters:');
-  console.log(response);
+async function main() {
+  const container = require('@google-cloud/container');
+
+  // Create the Cluster Manager Client
+  const client = new container.v1.ClusterManagerClient();
+
+  async function quickstart() {
+    const zone = 'us-central1-a';
+    const projectId = await client.getProjectId();
+    const request = {
+      projectId: projectId,
+      zone: zone,
+    };
+
+    const [response] = await client.listClusters(request);
+    console.log('Clusters:');
+    console.log(response);
```

Comment:
Maybe `console.log('Clusters: ', response);

Reply:
Fixed in https://github.com/googleapis/nodejs-cloud-container/pull/519/commits/641e197283d23fa573b4979a635c714fff6d3171